fleeto_backend/config/routes.rb

```diff
@@ -1,3 +1,22 @@
 Rails.application.routes.draw do
+
+  devise_for :customers
+
+  api_version(:module => "V1", :path => {:value => "v1"}) do
```

Comment:
Sounds reasonable. Both API are independent. 
Let's @loaighoraba let's discuss this.